Gemfile.lock

```diff
@@ -226,6 +226,3 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
   web-console (~> 2.0)
   whenever
-
-BUNDLED WITH
-   1.11.2
```

Comment:
(Placing this comment here as GitHub won't let me on the file itself)

Do we have licensing for the nature.jpg file? If so, what's the license?